src/h5web/vis-packs/core/heatmap/HeatmapVis.tsx

```diff
@@ -103,6 +106,7 @@ function HeatmapVis(props: Props) {
           scaleType={scaleType}
           alphaValues={alphaArray && (alphaArray.data as number[])}
           alphaDomain={alphaDomain}
+          flipYAxis={flipYAxis}
```

Comment:
Do you think this info should be retrieved from the axis system context (i.e. single source of truth)?

Reply:
I considered it as it would indeed feel cleaner. But then, I saw it required two destructurings so my laziness won:
```
  const { ordinateConfig } = useAxisSystemContext();
  const { flip: flipYAxis } = ordinateConfig;
```

Happy to implement it like that if you prefer though

Reply:
yeah the second one seems reasonable